### ISSUE TITLE
Full adder circuit not working

### ISSUE THREAD

**Issue by LadyCailin** [LadyCailin/TestIssueUploader#251]
CRAFTBOOK-2239 - Reported by Josh.Mann on 2012-11-08 23:55:57 UTC

When i apply a redstone signal nothing happens.

**Comment by LadyCailin** [comment]
Comment by me4502 on 2012-11-09 05:24:26 UTC

Are you able to explain what its supposed to do?

**Comment by LadyCailin** [comment]
Comment by Josh.Mann on 2012-11-09 23:43:28 UTC

well yes. When you turn 1 of the inputs on, the output should turn on. when you turn on another the carry out will turn on and output 1 will turn off. with that plug the carry out into another adder into the carry in input and so on. I know what this is because I've made an adder before out of just redstone, so i know the function of one. All the outputs that are on, then go to a binary to decimal or hexadecimal decoder then plugged into a display/any sort of know what the final # is. This is used in a calculator for example. I've made many of those too.
